backend/src/agents/tools/auth_tools.py

```
@@ -0,0 +1,537 @@
+            if creds and creds.get('selectors'):
+                selectors = creds.get('selectors')
+
+            print("[smart_login] falling back to credential-based login")
+            # If selectors not saved, extract them from parsed_html
+            if not selectors:
+                selectors = SelectorExtractorTool()._run(parsed_html)
+                # Save selectors for future
+                try:
+                    save_selectors(site_key, selectors)
+                except Exception:
+                    pass
```

Ruff (0.12.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)